lib/screenplay_web/auth_manager/auth_manager.ex

```diff
@@ -24,21 +26,26 @@ defmodule ScreenplayWeb.AuthManager do
 
   def resource_from_claims(_), do: {:error, :invalid_claims}
 
-  @spec claims_access_level(Guardian.Token.claims()) :: access_level()
+  @spec claims_access_level(Guardian.Token.claims()) :: list(access_level())
   def claims_access_level(%{"roles" => roles}) when not is_nil(roles) do
-    cond do
-      @screenplay_admin_role in roles ->
-        :emergency_admin
-
-      @screens_admin in roles ->
-        :screens_config_admmin
-
-      true ->
-        :read_only
+    access_levels =
+      []
+      |> append_if(@screenplay_emergency_admin_role in roles, :emergency_admin)
+      |> append_if(@screens_admin in roles, :screens_admin)
+      |> append_if(@pa_message_admin in roles, :pa_message_admin)
```

Comment:
It might be easier to see the correspondence between keycloak roles and, uh, auth manager roles? if we declared them all together like:

```elixir
@roles %{"screenplay-emergency-admin" => :emergency_admin, ...}
```

Then determining the access levels would be a breezier:

```elixir
Enum.map(roles, &Map.get(@roles, &1)) |> Enum.reject(&is_nil/1)
```

Reply:
Woah, that is slick. Definitely using that.